resources/zib/zib-BodyHeight.xml

```diff
@@ -0,0 +1,170 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-bodyheight-v3.1.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/BodyHeight-v3.1.1(2020EN)" />
+    <name value="zib BodyHeight-v3.1.1(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="BodyHeight" />
+      <alias value="Lichaamslengte" />
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.1" />
+        <comment value="BodyHeight" />
+      </mapping>
+    </element>
+    <element id="Observation.extension">
+      <path value="Observation.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.extension:position">
+      <path value="Observation.extension" />
+      <sliceName value="position" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/observation-bodyPosition" />
+      </type>
+    </element>
+    <element id="Observation.extension:position.value[x]">
+      <path value="Observation.extension.value[x]" />
+      <short value="Position" />
+      <definition value="The person's position during the measurement." />
+      <alias value="Positie" />
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.2.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.5" />
+        <comment value="Position" />
+      </mapping>
+    </element>
+    <element id="Observation.code.coding">
+      <path value="Observation.code.coding" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.code.coding:bodyHeightCode">
+      <path value="Observation.code.coding" />
+      <sliceName value="bodyHeightCode" />
+      <min value="1" />
+      <max value="1" />
+      <patternCoding>
+        <system value="http://loinc.org" />
+        <code value="8302-2" />
+      </patternCoding>
```

Comment:
FHIR specs

"The element Observation.code.coding is sliced based on the values of value:code, value:system"

I don't see a confirmed way that we do the "Vital signs" way in the guidelines: https://informatiestandaarden.nictiz.nl/wiki/FHIR:Vissue-Github-62_FHIR_Profiling_Guidelines_R4#HL7_Vital_Signs